pages/translations/de/messages.po

```diff
@@ -0,0 +1,3123 @@
+
+#: /content/amp-dev/documentation/guides-and-tutorials/optimize-measure/_blueprint.yaml
+msgid "Optimize & Measure"
+msgstr "Optimieren & Messen"
+
+#: /views/partials/footer.j2:41
+msgid "Overview"
+msgstr "Überblick"
+
+#: /content/amp-dev/about/stories.html
+#: /views/partials/case-grid.j2:14
+msgid "Parallax Storytelling in AMP"
+msgstr "Geschichten erzählen mit Parallax in AMP"
+
+#: /content/amp-dev/support/faq/platform-involvement.md
+msgid "Platform and Technology Company Involvement"
+msgstr "Beteiligung von Plattformen und Technologieunternehmen"
+
+#: /views/partials/footer.j2:63
+msgid "Platform and Vendor Partners"
+msgstr "Plattform- und Vertriebspartner"
+
+#: /content/shared/experiments.html:21
+msgid "Please reload the page."
+msgstr "Bitte laden Sie die Seite neu."
+
+#: /views/partials/search.j2:146 /views/partials/search.j2:155
+msgid "Please try again later."
+msgstr "Versuchen Sie es später erneut."
+
+#: /views/partials/search.j2:216
+msgid "Popular Articles"
+msgstr "Beliebte Artikel"
+
+#: /views/examples/preview.j2:19
+msgid "Preview"
+msgstr "Vorschau"
+
+#: /views/detail/docs-detail.j2:48
+msgid "Previous chapter"
+msgstr "Vorheriges Kapitel"
+
+#: /views/partials/footer.j2:83
+msgid "Privacy Policy"
+msgstr "Datenschutzerklärung"
+
+#: /content/amp-dev/support/faq/publisher-monetization.md
+msgid "Publisher Monetization"
+msgstr "Monetarisierung für Verlage"
+
+#: /views/partials/search.j2:116
+msgid "Results"
+msgstr "Ergebnisse"
+
+#: /views/partials/footer.j2:65
+msgid "Roadmap"
+msgstr "Roadmap"
+
+#: /content/amp-dev/about/stories.html
+msgid "SF Chronicle"
+msgstr "SF Chronicle"
+
+#: /views/partials/case-grid.j2:53
+msgid "Screenshot of use case:"
+msgstr "Screenshot des Anwendungsfalls:"
+
+#: /views/partials/search.j2:59
+msgid "Search"
+msgstr "Suche"
+
+#: /views/partials/case-grid.j2:66
+msgid "See all use cases"
+msgstr "Alle Anwendungsfälle anzeigen"
+
+#: /views/partials/teaser-grid.j2:135 /views/partials/teaser.j2:137
+msgid "See example"
+msgstr "Beispiel öffnen"
+
+#: /content/amp-dev/about/success-stories/index.html:14
+msgid "See what others have built"
+msgstr "Sehen Sie, was andere erstellt haben"
+
+#: /views/partials/header.j2:99
+msgid "Select a language"
+msgstr "Sprache wählen"
+
+#: /views/partials/rolling-formats.j2:3
+msgid "websites"
+msgstr "Websites"
+
+#: /views/partials/rolling-formats.j2:3
+msgid "stories"
+msgstr "Storys"
+
+#: /views/partials/rolling-formats.j2:3
+msgid "ads"
+msgstr "Anzeigen"
+
+#: /views/partials/rolling-formats.j2:3
+msgid "emails"
+msgstr "E-Mails"
+
+#: /views/partials/rolling-formats.j2:3
+msgid "email"
+msgstr "E-Mail"
+
+#: /content/amp-dev/index.html:65
+msgid "AMP is a web component framework to easily create <span style=\"white-space: nowrap;\">user-first</span><br>"
+msgstr "AMP ist ein Webkomponenten-Framework, mit dem auf einfache Weise folgende <span style=\"white-space: nowrap;\">nutzerorientierte</span><br> Dinge erstellt werden können:"
```

Comment:
Yep, that's one of the issues what we ran into, also with other languages. The way it is now, the rolling format section (website/stories/ads/emails) is always at the end of the sentence, which makes it hard/impossible to translated into some languages.
For those languages (including German) we did our best to rephrase for the line to make sense.
@patrickkettner Can this string somehow be restructured in a way that would use a variable? E.g. 
msgid "AMP is a web component framework to easily create  %s" - where %s would be replaced by the animation, so that its position could be changed?